src/pages/pupils/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/index.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  GetStaticPathsResult,
+  GetStaticProps,
+  GetStaticPropsResult,
+  NextPage,
+} from "next";
+import { oakDefaultTheme, OakThemeProvider } from "@oak-academy/oak-components";
+
+import { QuizEngineProvider } from "@/components/PupilJourneyComponents/QuizEngineProvider/QuizEngineProvider";
+import { QuizRenderer } from "@/components/PupilJourneyComponents/QuizRenderer/QuizRenderer";
+import getPageProps from "@/node-lib/getPageProps";
+import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
+import { PupilLessonOverviewData } from "@/node-lib/curriculum-api";
+import {
+  getFallbackBlockingConfig,
+  shouldSkipInitialBuild,
+} from "@/node-lib/isr";
+
+export type PupilLessonOverviewPageProps = {
+  curriculumData: PupilLessonOverviewData;
+};
+
+const Pupils: NextPage<PupilLessonOverviewPageProps> = ({ curriculumData }) => {
+  const { starterQuiz } = curriculumData;
+
+  return (
+    <OakThemeProvider theme={oakDefaultTheme}>
+      <QuizEngineProvider questionsArray={starterQuiz ?? []}>
+        <QuizRenderer />
+      </QuizEngineProvider>
+    </OakThemeProvider>
+  );
+};
+
+export type URLParams = {
+  lessonSlug: string;
+  unitSlug: string;
+  programmeSlug: string;
+};
+
+export const getStaticPaths = async () => {
```

Comment:
this looks like it's not doing anything right now. Is it just cut and paste from other pages ?